common/djangoapps/third_party_auth/samlproviderconfig/tests/test_samlproviderconfig.py

```diff
@@ -23,7 +23,8 @@
     'metadata_source': 'http://test.url',
     'name': 'name-of-config',
     'enabled': 'true',
-    'slug': 'test-slug'
+    'slug': 'test-slug',
+    'country': 'https:://example.customer.com/countrycode'
```

Comment:
is this value a valid value for country field? I would check the EnterpriseCustomer model maybe for ref

Reply:
yeah, unsure because I don't know what the expected user input is here. Inquired with the ECS team and will report back.

Reply:
I made the original example (realizing I added an extra colon) looking at Rob's example of 
```
{
"extra_field_definitions": [
{
"urn": "https://federation-sts.customer.com/schemas/claims/1/countrycode",
"name": "country"
}
]
}
```
Do you think this is unclear for devs looking at the test? I could add a comment stating that we're looking for a urn (I'm fairly certain it's a urn but still waiting for confirmation).